### DISCUSSION TITLE
How to get the correct transcriptions for Acronyms using coqui english model with custom language model

### DISCUSSION THREAD

**Discussion by rahulmunet** [coqui-ai/STT#2355] [Q&A]
Hello,
I am trying to create a voice assistant where the voice assistant will hear the acronyms and try to give out the possible fullforms. For this purpose i am training a custom language model with the possible acronyms. But the English model with this custom language model is not able to transcribe the acronyms correctly. Can you guide me how to get good accuracy of transcriptions for acronyms?

**Comment by FrontierDK** [comment]
The TTS doesn't have to be the only component in a system - use lists with known acronyms and do string replacements afterwards?
